src/priya_t/ProgrammingTest_2_LastEvenNofromString.java

```diff
@@ -0,0 +1,30 @@
+package priya_t;
+
+public class ProgrammingTest_2_LastEvenNofromString {
+	String str = "te3ch4nocred8its";
```

Comment:
What is the purpose of declaring string here, you already pass string as a paramter